src/compiler/config/test/validate-custom.spec.ts

```diff
@@ -26,6 +26,24 @@ describe('validateCustom', () => {
       },
     ];
     const { diagnostics } = validateConfig(userConfig, mockLoadConfigInit());
-    expect(diagnostics.length).toBe(1);
+    // TODO(STENCIL-1107): Decrement the right-hand side value from 2 to 1
+    expect(diagnostics.length).toBe(2);
+    // TODO(STENCIL-1107): Keep this assertion
+    expect(diagnostics[0]).toEqual({
+      header: 'Build Warn',
+      level: 'warn',
+      lines: [],
+      messageText: 'test warning',
+      type: 'build',
+    });
+    // TODO(STENCIL-1107): Remove this assertion
+    expect(diagnostics[1]).toEqual({
```

Comment:
We need to add this as [this field](https://github.com/ionic-team/stencil/pull/5269/files#diff-cfdd0f19925ea4e2870c4390ad870f820215b69acf2b4bba2eef548036fe95ebL102) is being used by our tests still. In adding the warning [here](https://github.com/ionic-team/stencil/pull/5269/files#diff-53c120bef3205f89229fbe98a5aadfcb51012ead144729ade2fc597c1e9ad5e9R270-R275), that caused an additional warning to trigger.

I figured it'd be best to just:
1. Increment the count of warns we assert against for now, since there's no other good way to 'split up' the number of assertions we get 
2. Add explicit assertions for the existing warning and here. That way, in STENCIL-1107, we can leave the first one and we're a little better off in this test suite than where we started